docs/source/models/risk.rst

```diff
@@ -4,17 +4,148 @@
 Modeling Risk Exposure
 ======================
 
-.. todo::
+.. note::
 
-   Overview of modeling GBD risk factor exposure.
+   Vivarium has the infrastructure to model any risks in the GBD hierarchy. 
+   You should use tools from ``vivarium_inputs`` to pull and examine the 
+   data for your risks before including them in your model.
 
-.. contents:
+GBD Overview
+------------
 
 What is a risk factor?
-----------------------
+++++++++++++++++++++++
+A risk factor is any attribute, characteristic or exposure of an individual
+that increases the likelihood of developing a disease or injury. In GBD, a risk factor refers to a exposure that leads to a loss of health in the population;
+it can be anywhere on the causal chain as long as there is evidence to show that the risk factor is causally linked to an outcome disease or injury.
+
+What do we measure?
```

Comment:
I think we should also consider 1) exposure of a risk (absolute, not a SEV) and 2) relationship between risk and outcome pairs (risk ratios) as things that are measured in GBD